OSC/ondemand
# New directory picker

This is phase 1 of the 1st Class File Picker.  At this point, the functionality simply includes navigating to and selecting a directory or file and placing it in the working directory text box.

There are no options passed in from the form to set certain functionality.  That will come in the next phase.

The best way to test this, is to

1. Checkout this branch. (newDirectoryPicker)
2. run bin/setup
3. Modify the form in bc_osc_codeserver in the following way.
```
Make sure you delete or rename the file form.js as it conflicts if you do not.
Change the widget name in form.yml.erb for working_dir to file_navigator
```
Now you can test the new file navigator in Code Server when selecting the working directory.
